exec.c

```diff
@@ -22,8 +22,14 @@ int executeCommand(char **argv)
 		char *cmd = argv[0];
 		char *envp[] = {NULL};
 
+		if (access(cmd, X_OK) != 0)
+			cmd = handlePath(cmd);
+
 		if (cmd == NULL)
```

Comment:
Remove the `exit()` because we don't want the shell to stop abruptly unless given special commands.